cmd/argocd/commands/utils/prompt_test.go

```diff
@@ -20,3 +20,30 @@ func TestConfirm_PromptsEnabled_False(t *testing.T) {
 	prompt := NewPrompt(false)
 	assert.True(t, prompt.Confirm("Are you sure you want to run this command? (y/n) "))
 }
+
+// Returns true, true when prompt is disabled
```

Comment:
nit:
```suggestion
// Returns true when prompt is disabled
```

Reply:
true, true means here that both booleans in response will be true
ConfirmAll returns two bolean flags, first mean that user confirmed , second that he confirmed all entities to be processed